### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
https://Zelinskyi-Serhii.github.io/layout_search-bar-airbnb/
https://Zelinskyi-Serhii.github.io/layout_search-bar-airbnb/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Awenir" ;
```
Comment (OleksiiBlezniuk) [inline src/style.css:6]:
```suggestion
  font-family: "Avenir" ;
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  font-family: "Awenir", sans-serif;
+  font-weight: 300;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:20]: Set font styles to `html` tag

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  font-family: inherit;
+  outline: none;
+}
+
+.search-form-small {
+  height: 42px;
+}
+
+.image {
+  display: block;
```
Comment (OleksiiBlezniuk) [inline src/style.css:50]: It's unnecessary rule

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  position: relative;
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.input::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+  color: #3d4e61;
```
Comment (OleksiiBlezniuk) [inline src/style.css:81]: use `inherit`

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
Please pay attention to code style, double check your solution with checklist

File: src/style.css (hunk 3 above)
Comment (Zelinskyi-Serhii) [reply src/style.css]: icon-display
I agree with you, i just copied from photo.

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**